docs/src/manual/construction.md

```diff
@@ -133,10 +133,19 @@ These are all examples of ill-formed intervals, resulting in the decoration `ill
 
 ### Guarantee
 
-A *guarantee* is yet another label, independent of decorations, and not described by the IEEE Standard 1788-2015 specifications. Its purpose is to accomodate for Julia's extensive conversion and promotion system, while retaining reliability in computations. Specifically, an interval `x` constructed via [`interval`](@ref) satisfies `isguaranteed(x) == true`. However, if a call to `convert(::Type{<:Interval}, ::Real)` occurs, then the resulting interval `x` satisfies `isguaranteed(x) == false`.
+A *guarantee* is yet another label, independent of decorations, and not described by the IEEE Standard 1788-2015 specifications. Its purpose is to accomodate for Julia's extensive conversion and promotion system, while retaining reliability in computations. Specifically, an interval `x` constructed via [`interval`](@ref) satisfies `isguaranteed(x) == true`. However, if a call to `convert(::Type{<:Interval}, ::Real)` occurs, then the resulting interval `x` satisfies `isguaranteed(x) == false`, receiving the 'NG' (not guaranteed) decoration. For instance, consider the following examples:
```

Comment:
One minor detail: perhaps instead of the word "decoration" it should be "label"?
Just because "decoration" has a very specific meaning in the IEEE Standard 1788-2015, and we ended up deciding that this "guarantee" flag ought to be an independent marker.

Reply:
Thank you, I made the change already.